styles/components/Events/Filters.module.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../variables.scss';
```

Comment:
@advaithg I don't think this look as you might want on phone. Not sure how much you care about that / if you wanted to review because of that!
![CAD9037E-442D-4109-9D98-3BDF792CACEC](https://user-images.githubusercontent.com/29292203/151056982-f86f28c9-1531-43b0-87dd-151113ab194c.png)

Reply:
Yeah I think this is not a great idea to have on mobile, so I see two options here:

1. Hide the filter option entirely on mobile
2. Fix it so search and filter buttons are underneath the search bar + centered

either way, I'd rather have a feature non-existent on mobile than not responsive
I think I'd prefer option 2, but if you would like to get this PR merged in, you can also do 1 and open a separate issue for 2 later!